Automattic/jetpack
# Masterbar: updates the default URL for Plans

#### Changes proposed in this Pull Request:

* Updates the default link in `Upgrades → Plans` to "Plans" instead of "My Plan".
* Updates tests accordingly.
* Related Calypso changes: https://github.com/Automattic/wp-calypso/pull/53884

#### Jetpack product discussion

p7DVsv-ago-p2#comment-36411

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* You can either:
  * Apply D63148-code in your sandbox to test this site in WordPress.com.
  * Switch to the branch of this PR in your local environment and test an Atomic site on WordPress.com.
* Ensure you're redirected to “Plans” when navigating into `Upgrades → Plans`.
